src/mixins/routable.js

```diff
@@ -25,7 +25,9 @@ export default {
     generateRouteLink () {
       let exact = this.exact
       let tag
-
+      const normalizedListeners = Object.keys(this.$listeners).map(event =>
+        Object.create({ [event.replace(/(&|!|~)/g, '')]: this.$listeners[event] })
```

Comment:
Why `Object.create()`?

The regex can just be `event.replace(/[&!~]/, '')`

Reply:
Uh, I was on a rush; couldn't find a way to return an object from an arrow function that time. But I fixed it know. 😅